### ISSUE TITLE
Spiky audio capture

### ISSUE THREAD

**Issue by Ma100Dev** [atomic14/esp32_audio#24]
Hello!
I am trying to listen to the audio recorded by the ESP32, but it just sounds like a bunch of clicks. I looked at the raw audio data with Audacity and the audio waveform indeed is almost exclusively formed out of spikes. What could be causing this and how would one go about fixing it? I experimented with whether lowering the gain on the MAX9814 module by tying its gain pin high (40dB gain), but it did not help.
![Waveform](https://i.imgur.com/AMGyGrA.png) |
|:--:|
| The raw audio data in Audacity |

**Comment by frmunozForcast** [comment]
@Ma100Dev I know this is an old issue, but I'm facing the exact same problem, did you managed to solve it?

**Comment by Ma100Dev** [comment]
No. There was never an answer to this issue and I didn't manage to figure it out myself.
